connectors/uplynk/src/main/java/com/theoplayer/android/connector/uplynk/UplynkConnector.kt

```diff
@@ -21,7 +21,7 @@ internal const val TAG = "UplynkConnector"
  * @see [Uplynk Media Platform](https://docs.edgecast.com/video/index.html)
```

Comment:
https://docs.edgecast.com/ is 404ing now too? Seriously, where are customers supposed to find *anything* about Uplynk nowadays? 🙄